bash_completion

```diff
@@ -2078,14 +2078,25 @@ _command_offset()
                 # complete -F <function>
 
                 # get function name
-                local func=${cspec#*-F }
-                func=${func%% *}
+                local func=${cspec#*-F } newcspec
```

Comment:
In this PR, the new completion spec is assigned to `newcspec` and not reflected in `cspec`. However, it seems `cspec` is later tested for whether it contains `-o xxxx`. Do you think we should still perform this check on the older `cspec` and skip the check for the new completion spec? Maybe we can directly update `cspec`?

Reply:
This required a little bit of digging to find out what this is required for. I found some info on [this](https://alioth-lists-archive.debian.net/pipermail/bash-completion-devel/2011-September/003906.html) mailing list. Seems like the old mount compspec relied on `-o dirname` if the completion function didn't generate matches which thus must also be set during meta-completion to work. I think running this for the new compspec only should be sufficient.

Reply:
OK!